examples/kitty/board/odroidc4/kitty.system

```diff
@@ -329,7 +329,7 @@
 
     <channel>
         <end pd="i2c_driver" id="0" />
-        <end pd="i2c_virt" id="1" />
+        <end pd="i2c_virt" id="2" />
```

Comment:
What was the need for this change? Was it something to do with the sddf i2c changes?